release-notes/8.0/preview/api-diff/preview4/Microsoft.NETCore.App/8.0-preview4_System.Threading.Tasks.md

```diff
@@ -0,0 +1,28 @@
+# System.Threading.Tasks
```

Comment:
`System.Threading.Tasks` LGTM.

cc. @dotnet/area-system-threading-tasks, @stephentoub

Reply:
The serialization diffs on the last two APIs have the same question as https://github.com/dotnet/core/pull/8469/files#r1198216476 on if anything changed here.